spiel/st/m0t1fs_spiel_sns_common_inc.sh

```diff
@@ -131,28 +131,28 @@ spiel_prepare()
 	export SPIEL_OPTS=$SPIEL_OPTS
 	export SPIEL_FIDS_LIST=$SPIEL_FIDS_LIST
 
-	echo SPIEL_OPTS=$SPIEL_OPTS
-	echo SPIEL_FIDS_LIST=$SPIEL_FIDS_LIST
+	echo SPIEL_OPTS="$SPIEL_OPTS"
+	echo SPIEL_FIDS_LIST="$SPIEL_FIDS_LIST"
 
 	# install "motr" Python module required by m0spiel tool
-	cd $M0_SRC_DIR/utils/spiel
+	cd "$M0_SRC_DIR"/utils/spiel
 	python3 setup.py install --record $PYTHON_STUFF > /dev/null ||\
 		die 'Cannot install Python "motr" module'
 	cd -
 }
 
 spiel_cleanup()
 {
-	cd $M0_SRC_DIR/utils/spiel
+	cd "$M0_SRC_DIR"/utils/spiel
 	cat $PYTHON_STUFF | xargs rm -rf
 	rm -rf build/ $PYTHON_STUFF
 	cd -
 }
 
 spiel_sns_repair_start()
 {
-echo $M0_SRC_DIR/utils/spiel/m0spiel $SPIEL_OPTS
-    $M0_SRC_DIR/utils/spiel/m0spiel $SPIEL_OPTS <<EOF
+echo "$M0_SRC_DIR"/utils/spiel/m0spiel "$SPIEL_OPTS"
+    "$M0_SRC_DIR"/utils/spiel/m0spiel "$SPIEL_OPTS" <<EOF
```

Comment:
$SPIEL_OPTS should not be double quoted.

Reply:
done